### PR TITLE
Restores gibbing for suicide C4

### PR DESCRIPTION
**Pull request by ghost**
:boom: :boom: :boom: :boom: :boom: :boom:
Addresses #8112 
Also adds revolutionary suicides + fixes "NARSIE" because lack of hyphen is cancer

### CODE REVIEW

**Comment by Dorsisdwarf** [comment]
```
else if(role == "wizard")
	message_say = "THIS WAS A TERRIBLE IDEA!"

```

File: code/game/objects/items/weapons/explosives.dm [hunk 1]
```diff
@@ -33,11 +33,14 @@
...
-				message_say = "FOR NARSIE!"
+				message_say = "FOR NAR-SIE!"
+			else if(role == "revolutionary" || role == "head revolutionary")
+				message_say = "VIVA LA REVOLUTION!"
 	user.say(message_say)
 	target = user
 	message_admins("[key_name(user, user.client)](<A HREF='?_src_=holder;adminmoreinfo=\ref[user]'>?</A>) suicided with [src.name] at ([x],[y],[z] - <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[x];Y=[y];Z=[z]'>JMP</a>)",0,1)
 	explode(get_turf(user))
+	user.gib()
 	return .
```
Comment (RemieRichards) [inline code/game/objects/items/weapons/explosives.dm]: remove this return and the . on this line, it's entirely unnecessary and is technically (no matter how small) adding overhead.

**Comment by RandomMarine** [comment]
Could you add a gang line, too?

**Comment by Xhuis** [comment]
Got rid of the return and added a wiz quote. Not gonna do gang because it's a dead gamemode

**Comment by Xhuis** [comment]
Wiz quote broke it, so I removed it :book:

**Comment by Atskadan** [comment]
make the chaplain say "FOR [respective deity]"

**Comment by Cheridan** [comment]
since it's no longer returning . then I think the . = BRUTELOSS at line 25 at the top should get taken out as well.

**Comment by RemieRichards** [comment]
@Cheridan ```.``` is always returned, which is why I asked the ```return .``` to be removed, adding the "return" statement actually adds overhead over the default ```.``` return.

**Comment by Cheridan** [comment]
I know?? I didn't say to readd the return statement, I said to remove the line where . is being set to BRUTELOSS since it shouldn't be necessary anymore

**Comment by RemieRichards** [comment]
Why won't it be necessary? because of the gibbing? I guess.
(Also make yourself clearer next time please, my response was perfectly valid for how your comment seemed)

**Comment by Dorsisdwarf** [comment]
Can you make a sleep(10) or something on this, just so that people around you have a chance to register that you're going to selfdestruct?

**Comment by Xhuis** [comment]
Squashed commits, added a sleep, and removed the unnecessary returns.

**Comment by Atskadan** [comment]
>make the chaplain say "FOR [respective deity]"

pls

**Pull request merged**